docs/Development/working-with-git.md

```diff
@@ -20,15 +20,15 @@ In general, if you are only going to be using the model and not working directly
 If you plan on contributing to model development or would like a systematic way to incorporate updates to the VIC source code, we encourage you to use Git.  The following sections are designed to get you started using git and working with the VIC source code repository.
 
 ## Git Resources
-If you are not familiar with [Git](http://git-scm.com/) yet, we encourage you to spend a few minutes getting antiquated with the system before you starting working with the VIC source code and Git.  It's not difficult to use and a few minutes of learning about [Git](http://git-scm.com/) will go along way in helping you manage your code development.
+If you are not familiar with [Git](http://git-scm.com/) yet, we encourage you to spend a some time becoming acquainted with the system before you starting working with the VIC source code and Git.  A few minutes of learning about [Git](http://git-scm.com/) will go along way in helping you manage your code development.
```

Comment:
"we encourage you to spend a some time becoming acquainted with the system before you starting working with the VIC source code and Git."

suggested change

"we encourage you to get acquainted with it before you starting working with the VIC source code."